pkg/controller/stacks/stack/stack.go

```diff
@@ -734,6 +748,8 @@ func (h *stackHandler) processJob(ctx context.Context) error {
 	// ensure the job is set to use this stack's service account that we created
 	jobSpec := *controllerJob.Spec.DeepCopy()
 	jobSpec.Template.Spec.ServiceAccountName = h.ext.Name
+	name := h.ext.Name + "-job"
```

Comment:
i've been wondering if we need to keep maintaining this "job" (instead of deployment) support. i don't think it's really used anywhere at all, it was sort of theoretical when we first added it, right? something to think about more, i keep questioning it every time i see PRs having to match a change to this code path too, which increases our maintenance burden for functionality that I don't think is used at all 🤔

Reply:
Stacks have moved away from the possible solutions jobs support was made to provide. I'll create an issue to remove them.

Reply:
The issue is #1249